domain/src/androidTest/java/com/github/leuludyha/domain/useCases/MixedUseCasesTest.kt

```diff
@@ -0,0 +1,128 @@
+package com.github.leuludyha.domain.useCases
+
+import android.content.Context
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import com.github.leuludyha.domain.MockLocalFailureRemoteSuccessLibraryRepositoryImpl
+import com.github.leuludyha.domain.MockLocalRemoteFailureLibraryRepositoryImpl
+import com.github.leuludyha.domain.MockLocalSuccessLibraryRepositoryImpl
+import com.github.leuludyha.domain.MockTrueNetworkProvider
+import com.github.leuludyha.domain.model.library.Mocks.authorRoaldDahl
+import com.github.leuludyha.domain.model.library.Mocks.editionMrFox
+import com.github.leuludyha.domain.model.library.Mocks.workMrFox
+import com.github.leuludyha.domain.model.library.Result
+import com.github.leuludyha.domain.useCase.*
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class MixedUseCasesTest {
+    lateinit var context: Context
+    @Before
+    fun setup() {
+        context = InstrumentationRegistry.getInstrumentation().targetContext
+    }
+
+    @OptIn(ExperimentalCoroutinesApi::class)
```

Comment:
I believe you can save some time and lines by putting this on the whole class instead of each test.